http/src/androidMain/kotlin/com/mirego/trikot/http/android/requestFactory/KtorHttpRequestFactory.kt

```diff
@@ -24,23 +25,33 @@ import io.ktor.content.TextContent
 import io.ktor.http.ContentType
 import io.ktor.http.HttpMethod
 import io.ktor.util.flattenEntries
+import kotlin.coroutines.CoroutineContext
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 import org.reactivestreams.Publisher
-import kotlin.coroutines.CoroutineContext
+
+private const val DEFAULT_TIMEOUT_MS = 10000L
```

Comment:
As mentionned in [my other comment](https://github.com/mirego/trikot.http/pull/46/files#r603395706) I would use a [Duration](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration/) here.